Geometry/DTGeometryBuilder/plugins/dd4hep/DTGeometryESProducer.cc

```diff
@@ -137,8 +138,6 @@ std::shared_ptr<DTGeometry> DTGeometryESProducer::produce(const MuonGeometryReco
   // Called whenever the alignments or alignment errors change
   //
   if (m_applyAlignment) {
-    // m_applyAlignment is scheduled for removal.
-    // Ideal geometry obtained by using 'fake alignment' (with m_applyAlignment = true)
     edm::ESHandle<Alignments> globalPosition;
     record.getRecord<GlobalPositionRcd>().get(m_alignmentsLabel, globalPosition);
```

Comment:
Sidetracking a bit, but I want to note that these calls need to be changed to use the `ESGetToken` to get the products. Getting products by labels in was disabled already some time ago in ESProducers.

Reply:
@makortel I made an issue about this problem: #31288